plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/BuildImageParams.java

```diff
@@ -40,7 +43,7 @@ public BuildImageParams withRepository(String repository) {
 
     /**
      * @param authConfigs
-     *         authentication configuration for private registries. Can be null
+     *         authentication configuration for registries. Can be null
```

Comment:
It doesn't make sense to write here that it can be null, so I suggest we need to move javadocs to getter. In that case description will fit purpose of method better.
Also use @Nullable instead of **Can be null** in that case (with getter).